lib/cfg.py

```diff
@@ -251,6 +253,83 @@ def cfg_version(filename):
         cfg[section] = d
     return cfg
 
+def cfg_altiumWorkspace(options):
+    """
+    ricerca del nome di tutti i progetti all'interno del file Workspace
```

Comment:
Gli online docs, del python sono quelle stringe che appaiono quando chiami la funzione help(), e ti fa vedere solo quella subito dopo la definizione della funzione.
Quindi in generale le descrizioni devono essere per l'utente che le usa, e quindi devono spiegare cosa fa la funzione e quali sono i sui parametri di ingresso. Ovviamente non importa avare chilometri di documentazione..